README.md

```diff
@@ -1,15 +1,12 @@
-##### **NOTE:** `dali_backend` is available in `tritonserver-20.11` and later
-
- :exclamation: **IMPORTANT** :exclamation:
+# DALI TRITON Backend
 
+<!---->
 `dali_backend` is new and rapidly growing. Official `tritonserver` releases might be behind
```

Comment:
@szalpal Is it still new and rapidly growing?

Reply:
👍🏻 Thank you for pointing that out.